resources/fetching/fetcher.go

```diff
@@ -53,6 +53,7 @@ const (
 	AccessAnalyzers           = "aws-access-analyzers"
 
 	GcpMonitoringType = "gcp-monitoring"
+	GcpServiceUsage   = "gcp-service-usage"
```

Comment:
we've kept all other types in in the gcp assets fetcher folder, consider including these two with the rest